nextjs/services/events/eventNewMentions.ts

```diff
@@ -18,7 +26,19 @@ export async function eventNewMentions({
     });
 
     if (user?.auth?.id) {
-      await pushUserMention({ userId: user.auth.id, threadId, channelId });
+      // we could save the mention type in the db instead
```

Comment:
We should store the mention type

Reply:
yup, I agree, going to do that in the second pass